server/src/test/scala/actors/persistent/staffing/SplitUtilSpec.scala

```diff
@@ -0,0 +1,104 @@
+package actors.persistent.staffing;
+
+
+import drt.shared._
+import org.specs2.mutable.Specification
+import uk.gov.homeoffice.drt.ports.Terminals.T1
+import uk.gov.homeoffice.drt.time.SDate
+
+class SplitUtilSpec extends Specification {
+
+  "SplitUtil" >> {
+    "split a shift into 14 minutes interval shifts" >> {
+      val startTime = SDate(s"2017-01-01T07:00").millisSinceEpoch
+      val endTime = SDate(s"2017-01-01T12:00").millisSinceEpoch
```

Comment:
And again here. We only need to check a 30 minute shift is cut into 2 15 minute shifts